demosplan/DemosPlanCoreBundle/ResourceTypes/ParagraphVersionResourceType.php

```diff
@@ -41,6 +41,18 @@ public function isAvailable(): bool
         return true;
     }
 
+    /**
+     * Emulate this behaviour.
+     *
+     * @see \demosplan\DemosPlanCoreBundle\Logic\Statement\StatementService::getEntityVersions
+     * @see \demosplan\DemosPlanCoreBundle\Logic\Document\ParagraphService::createParagraphVersion
+     */
+    public function isCreateAllowed(): bool
+    {
+        // @todo update to proper conditions
```

Comment:
@breierDemos I removed it d5bb78746b0e407bba65149e8e824fec7ea7c977 because ParagrapVersion is created by the StatementResourceType. In this way, we protect the Versions to be created manually as they are a copy of an existing entity